src/status_im/contexts/wallet/add_account/create_account/events.cljs

```diff
@@ -99,7 +100,7 @@
 
 (defn clear-create-account-data
   [{:keys [db]}]
-  {:db (update-in db [:wallet :ui :create-account] dissoc :new-keypair)})
+  {:db (update-in db [:wallet :ui :create-account] dissoc :new-keypair :workflow)})
```

Comment:
`dissoc` doesn't dive in nested maps, so here data will be removed under `:new-keypair`, not under `[:new-keypair :workflow]`

Reply:
Thx!